tests/puppeteer/pages/Install/install.js

```diff
@@ -0,0 +1,149 @@
+    this.termsConditionsCheckbox = '#set_license';
+
+    // Selectors for step 3
+    this.thirdStepPageTitle = '#sheet_system h2';
+    this.stepFinishedLeftMenu = '#leftpannel #tabs li.finished:nth-child(3)';
+
+    // Selectors for step 4
+    this.fourthStepPageTitle = '#infosShopBlock h2';
+    this.shopNameInput = '#infosShop';
+    this.countrySelect = '#infosCountry';
+    this.firstNameInput = '#infosFirstname';
+    this.lastNameInput = '#infosName';
+    this.emailInput = '#infosEmail';
+    this.passwordInput = '#infosPassword';
+    this.repeatPasswordInput = '#infosPasswordRepeat';
+
+    // Selectors for step 5
+    this.fifthStepPageTitle = '#dbPart h2';
+    this.dbLoginInput = '#dbLogin';
+    this.dbPasswordInput = '#dbPassword';
+    this.tablePrefixInput = '#db_prefix';
+    this.testDbConnectionButton = '#btTestDB';
+    this.createDbButton = '#btCreateDB';
+    this.dbResultCheckOk = '#dbResultCheck.okBlock';
+
+    // Selectors for Final step
+    this.finalStepPageTitle = '#install_process_success h2';
+    this.installationProgressBar = '#install_process_form #progress_bar .installing';
+    this.discoverFoButton = '#foBlock';
+
+    // Selectors in FO
+    this.FOLogo = '#_desktop_logo';
+  }
+
+  /**
+   * To check each step title
+   * @param selector, where to get actual title
+   * @param pageTitle, expected title
+   */
+  async checkStepTitle(selector, pageTitle) {
+    await this.page.screenshot({path: 'test.png'});
+    const title = await this.getTextContent(selector);
+    await global.expect(title).to.equal(pageTitle);
+  }
+
+  /**
+   * Change install language in step 1
+   */
+  async setInstallLanguage() {
+    await this.page.select(this.languageSelect, global.INSTALL_LANGUAGE);
+  }
+
+  /**
+   * Go to next step
+   */
+  async nextStep() {
+    await this.page.waitForSelector(this.nextStepButton, {visible: true});
+    await this.page.click(this.nextStepButton, {waitUntil: 'domcontentloaded'});
+  }
+
+  /**
+   * Click on checkbox to agree on terms and conditions if its not checked already in step 2
+   */
+  async agreeToTermsAndConditions() {
+    const isChecked = await this.elementChecked(this.termsConditionsCheckbox);
+    if (!isChecked) {
+      await this.page.click(this.termsConditionsCheckbox);
+    }
+  }
+
+  /**
+   * Fill Information and Account Forms in step 4
+   */
+  async fillInformationForm() {
+    await this.page.type(this.shopNameInput, global.SHOPNAME);
+    await this.page.select(this.countrySelect, global.INSTALL_COUNTRY);
+    await this.page.type(this.firstNameInput, 'demo');
+    await this.page.type(this.lastNameInput, 'demo');
+    await this.page.type(this.emailInput, global.EMAIL);
+    await this.page.type(this.passwordInput, global.PASSWD);
+    await this.page.type(this.repeatPasswordInput, global.PASSWD);
+  }
+
+  /**
+   * Fill Database Form in step 5
+   */
+  async fillDatabaseForm() {
+    await this.page.click(this.dbLoginInput, {clickCount: 3});
```

Comment:
Any reason you force click 3 times ?

Reply:
to clear the input before typing our value